ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/ThankYou.feature

```diff
@@ -14,3 +14,34 @@ Scenario: Representative participant has camera and microphone on laptop or desk
 	Then Representative should be on 'questionnaire completed' screen
 	When proceeds to next page
 	Then Representative should be on 'check your computer' screen
+
+@smoketest @VIH-4578
+Scenario: Representative views thank you page after submitting self test questionnaire completed page for a client who is suitable for a video hearing
```

Comment:
This test has a few too many steps I think. Would it be possible to reduce them somehow and make it more to the point of what it is testing?

Reply:
Reduced number of tests also merged two scenarios into one